### PR TITLE
[I AM BORED, THIS WILL BE FINISHED LATER!!] Adds What I'd Call SqueezyChem

### PR DESCRIPTION
**Pull request by Anderson-Cooper**
## About The Pull Request

Adds some chems that are useful for making poisons, meds, or drugs

## Why It's Good For The Game

https://www.youtube.com/watch?v=4wSBmbiStOU

## Changelog
:cl:
add: Adds some pretty funny chems
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
I will literally ban you from repo if you keep doing this.
You know what I am talking about.

**Comment by SqueezyS** [comment]
> I will literally ban you from repo if you keep doing this.
> 
> You know what I am talking about.

There's a multiple things you could be talking about so I'll fix em all

**Comment by SqueezyS** [comment]
I unknowingly made a name that's actual irl chemistry

File: tgstation.dme [hunk 1]
```diff
@@ -3794,3 +3796,20 @@
 #include "ModularTegustation\tegu_xenobio_blackcross\black_extracts.dm"
 #include "ModularTegustation\tegu_xenobio_blackcross\slime_code.dm"
 // END_INCLUDE
+
+
+//░░░░▄▄▄▄▀▀▀▀▀▀▀▀▄▄▄▄▄▄
```
Comment (Kitsunemitsu) [inline tgstation.dme:3801]: Not professional, but very funny.
For now this can stay but has to be removed before merge.

Comment (SqueezyS) [reply tgstation.dme:3801]: :(

**Comment by SqueezyS** [comment]
I will finish this when I can, I will stop it for now

If anyone wants to make anything then please commit

I am bored of this

**Pull request closed without merging**